includes/validation/class-amp-validation-manager.php

```diff
@@ -245,6 +246,20 @@ function( $query_vars ) {
 			add_action( 'all_admin_notices', [ __CLASS__, 'print_plugin_notice' ] );
 
 			add_action( 'admin_bar_menu', [ __CLASS__, 'add_admin_bar_menu_items' ], 101 );
+
+			add_action(
+				'admin_bar_init',
+				static function () {
+					wp_enqueue_style(
+						'amp-icons',
+						amp_get_asset_url( 'css/amp-icons.css' ),
+						[ 'dashicons' ],
```

Comment:
```suggestion
						[ 'admin-bar', 'dashicons' ],
```

Reply:
Done in https://github.com/ampproject/amp-wp/pull/4382/commits/335e863ea78dbdeb69abae8cfa26bee7b2844ede.